src/common/components/ImageUpload/ImageUpload.vue

```diff
@@ -8,9 +8,14 @@ import { useI18n } from 'vue-i18n'
  */
 interface ImageUploadProps {
   /**
-   * Default image displayed in the left part of the file upload (before any upload)
+   * Default image name displayed in the left part of the file upload (before any upload)
    */
-  defaultImage: string
+  defaultImage?: string
```

Comment:
Si ce n'est plus que pour l'affichage du nom, il faut que ce soit clair

```suggestion
  defaultImageName?: string
```